reference/format/system_configuration.rst

```diff
@@ -0,0 +1,125 @@
+System Configuration
+====================
+
++-----------+------------------------------+
+| Filename  | ``system_configuration.yml`` |
```

Comment:
`system_configuration.yml` can contain `api_tree` block. With this block, the developer can add parameters what can be get via API.
An example you can find here https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/ActivityListBundle/Resources/config/system_configuration.yml#L67.